From 17053c4497e1ff64cd8ad62d84bbf4d3d496d154 Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Wed, 1 Apr 2020 17:24:54 +0800
Subject: [PATCH] ARM: dts: rv11xx-evb-v10: set dovdd boot on

Some boards can't get the camera sensor ID at one time,
So temporarily set dovdd boot-on to work around.

Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
Change-Id: I5a1aee1f930ed82dab221c71756b74fd032368e1
---
 arch/arm/boot/dts/rv11xx-evb-v10.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
index 1cb854a25219..c13d570c8b83 100644
--- a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
+++ b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
@@ -672,6 +672,7 @@
 			};
 
 			vcc_dovdd: LDO_REG5 {
+				regulator-boot-on;
 				regulator-min-microvolt = <1800000>;
 				regulator-max-microvolt = <1800000>;
 				regulator-name = "vcc_dovdd";
-- 
2.35.3

